guides/guides.json

```diff
@@ -102,6 +102,11 @@
           "name": "Integrate Existing Apps",
           "url": "./workflow/integrate_with_existing_apps.md",
           "isMdFile": true
+        },
+        {
+          "name": "Testing",
```

Comment:
You need to add it to the documentation.yaml file for it to appear in the new docs. This file can actually be deleted.

Reply:
Thanks, I removed it as part of this PR so it won't confuse others in the future.